src/riak_cs_gc_batch.erl

```diff
@@ -101,8 +102,8 @@ init([#gc_batch_state{
             %% EndKey <= BatchStart - Leeway
             _ = lager:info("Starting garbage collection: "
                            "(start, end) = (~p, ~p), "
-                           "leeway=~p, batch_start=~p, max_workers=~p",
-                           [StartKey, EndKey, Leeway, BatchStart, MaxWorkers]),
+                           "leeway=~p, batch_start=~p, max_workers=~p, batch_size=~p",
```

Comment:
Log messages is a kind of end user / operator interface, too. It's better to use the same word here as command line option.